antenna/throttler.py

```diff
@@ -28,6 +28,20 @@
 RESULT_TO_TEXT = {0: "ACCEPT", 1: "DEFER", 2: "REJECT", 3: "FAKEACCEPT"}
 
 
+def safe_get(data, key, default=""):
+    """Return data[key] or empty string.
+
+    Throttle rule conditions operate on strings since strings are the predominant value
+    type for crash report annotations.
+
+    This forces the value to be a string. For example, `None` will become `"None"`.
+
+    :return: string value
+
+    """
+    return str(data.get(key, default))
```

Comment:
Having a single function that does this is easier than adjusting the code everywhere. This is a sanitizing function, so as we hit issues with new and exciting data problems, we can tweak this function accordingly.